sycl/include/CL/sycl/detail/pi.h

```diff
@@ -259,7 +260,15 @@ typedef enum {
   PI_DEVICE_INFO_USM_CROSS_SHARED_SUPPORT =
       CL_DEVICE_CROSS_DEVICE_SHARED_MEM_CAPABILITIES_INTEL,
   PI_DEVICE_INFO_USM_SYSTEM_SHARED_SUPPORT =
-      CL_DEVICE_SHARED_SYSTEM_MEM_CAPABILITIES_INTEL
+      CL_DEVICE_SHARED_SYSTEM_MEM_CAPABILITIES_INTEL,
+  // These are Intel-specific extensions.
+  PI_DEVICE_INFO_PCI_ADDRESS = 0x10020,
```

Comment:
Why is this starting with offset 0x20, not 0x01?
Other extensions will need to start after these ones, is there a way we could some how force/verify this?

Reply:
There was a conflict with 0x10004 (CL_KHRONOS_VENDOR_ID_CODEPLAY), so I started the sequence later.

Reply:
I am OK to start with 0x100020, but how do we know there are no other overlaps (and will be no in future)?

Reply:
I am going to approve this as is, but would like to clarify how we can guarantee that future extensions of OpenCL don't overlap with future extensions of PI, while the two evolve pretty much independently. @jbrodman any comments on this?